mapper/dm_agent/src/monitor.rs

```diff
@@ -21,12 +21,12 @@ use std::time::Duration;
 
 #[derive(Debug)]
 pub struct DeviceMonitorConfig {
-    host: &'static str,
-    port: u16,
-    mqtt_client_id: &'static str,
-    mqtt_source_topic: &'static str,
-    mqtt_target_topic: &'static str,
-    batching_window: u64,
+    pub host: &'static str,
+    pub port: u16,
+    pub mqtt_client_id: &'static str,
+    pub mqtt_source_topic: &'static str,
+    pub mqtt_target_topic: &'static str,
+    pub batching_window: u64,
```

Comment:
This seems unrelated to that PR

Reply:
Same as above, while I create the config with the port number, it throws an error telling these struct members are private and cant update.

Reply:
Implemented a setter function and reverted them back to private